### PR TITLE
Localized build and loc fixes

### PR DESCRIPTION
**Pull request by MikhailArkhipov**
https://github.com/Microsoft/RTVS-pr/pull/20 for the setup

### CODE REVIEW

File: src/Common/Core/Impl/Microsoft.Common.Core.csproj [hunk 1]
```diff
@@ -16,25 +16,7 @@
...
-  </PropertyGroup>
-  <PropertyGroup>
-    <DefineConstants Condition="'$(VisualStudioVersion)'=='14.0'">VS14;$(DefineConstants)</DefineConstants>
-    <DefineConstants Condition="'$(VisualStudioVersion)'=='15.0'">VS15;$(DefineConstants)</DefineConstants>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Release|AnyCPU'">
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-  </PropertyGroup>
   <Import Project="..\..\..\R.Settings.targets" />
-  <PropertyGroup>
```
Comment (AlexanderSher) [inline src/Common/Core/Impl/Microsoft.Common.Core.csproj]: I don't remember what was the issue, but for some reason I had to keep these property groups inside the project. Either nuget or solution explorer.
Can you try to delete your .nuget folder in the appdata and build the whole thing from the empty enlistment?

Comment (MikhailArkhipov) [inline src/Common/Core/Impl/Microsoft.Common.Core.csproj]: It builds in the lab OK but I can try.

File: src/R.Build.Loc.targets [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <OutputFileExtension Condition="'$(OutputType)' == 'Library'">dll</OutputFileExtension>
+    <OutputFileExtension Condition="'$(OutputType)' != 'Library'">exe</OutputFileExtension>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FilesToLocalize Include="$(OutputPath)\$(AssemblyName).$(OutputFileExtension)">
+      <!--Required: Files to be localized -->
+      <TranslationFile>$(SourceDirectory)loc\lcl\{Lang}\$(AssemblyName).lcl</TranslationFile>
```
Comment (AlexanderSher) [inline src/R.Build.Loc.targets]: Are we going to add these files in a separate commit?

Comment (MikhailArkhipov) [inline src/R.Build.Loc.targets]: They will be coming from loc team AFAIK

File: src/Common/Core/Impl/Microsoft.Common.Core.csproj (hunk 1 above)
Comment (MikhailArkhipov) [inline src/Common/Core/Impl/Microsoft.Common.Core.csproj]: Seems to build OK

**Comment by AlexanderSher** [comment]
👍

**Comment by int19h** [comment]
👍

**Pull request merged**